### PR TITLE
Use freeze from ftw.testing in our tests.

### PR DESCRIPTION
**Pull request by mbaechtold**
This pull requests makes testing of the default news date more robust.

### CODE REVIEW

File: sources.cfg [hunk 1]
```diff
@@ -9,3 +9,8 @@ auto-checkout =
     ftw.simplelayout
 # https://github.com/4teamwork/ftw.datepicker/pull/3
     ftw.datepicker
+# https://github.com/4teamwork/ftw.testing/pull/21
```
Comment (mbaechtold) [inline sources.cfg]: :construction:

File: sources.cfg [hunk 2]
```diff
@@ -9,3 +9,8 @@ auto-checkout =
...
+    ftw.testing
+
+[branches]
+ftw.testing = jone-freeze-and-commit
```
Comment (mbaechtold) [inline sources.cfg]: :construction:

**Comment by mbaechtold** [comment]
The test should be passing, once there is a new release of `forbidden fruit`: https://github.com/clarete/forbiddenfruit/issues/14

File: sources.cfg [hunk 3]
```diff
@@ -9,3 +9,9 @@ auto-checkout =
     ftw.simplelayout
 # https://github.com/4teamwork/ftw.datepicker/pull/3
     ftw.datepicker
+# https://github.com/4teamwork/ftw.testing/pull/21
+    ftw.testing
+
+
+[branches]
+ftw.testing = jone-forbiddenfruit
```
Comment (mbaechtold) [inline sources.cfg]: :construction:

Comment (jone) [inline sources.cfg]: @mbaechtold merged :wink:

Comment (jone) [inline sources.cfg]: and released

Comment (mbaechtold) [inline sources.cfg]: Cool, thank you @jone.

I've updated this pull request. The extras `test` now requires ftw.testing >= 1.10.4.

**Comment by mbaechtold** [comment]
@jone, this is ready now!

**Pull request merged**